tox.ini

```diff
@@ -74,62 +79,12 @@ commands =
         -m pytest -v --tb native -s {posargs} {[vars]tests_path}/unit
     poetry run coverage report
 
-[testenv:integration]
+[testenv:integration-{provider,password-rotation,tls,upgrade,ha}]
```

Comment:
**praise** many thanks! it makes things way more maintainable and clear!

Reply:
+1. Really like it 👍🏾